readme.md

```diff
@@ -112,22 +119,24 @@ $ npm -v
 * **Question:** What gets logged? Why?
 
 * **Your Answer:**
+'charlotte' - because we exported the string from the profile.js file and imported it into our index file. Then when we ran our start command (which runs the index.js file) it does what we asked it to do in profile.js
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
 * **Your Answer:**
-
+`module.exports` -
+`require` - It's a function that takes a string that is a path to a file.
```

Comment:
* `module.exports` is by default an object. Whatever you assign to it will be what is pulled in when you `require()` elsewhere.